l10n_it_fatturapa_out/wizard/wizard_export_fatturapa_view_regenerate.xml

```diff
@@ -10,9 +11,11 @@
                 <attribute name="invisible">1</attribute>
             </xpath>
             <xpath expr="//button[@name='exportFatturaPA']" position="after">
-                <button name="exportFatturaPARegenerate" string="Re-Export"
-                        type="object" class="oe_highlight"/>
+                <button name="exportFatturaPARegenerate" string="Re-Export" type="object" class="oe_highlight"/>
```

Comment:
Sarebbe meglio non modificare parti non collegate alla specifica funzionalità oggetto della PR

Reply:
@eLBati aiutami a dire esticazzi? a volte fatico a capire il senso ..

Reply:
Il commento è mirato da una parte a facilitare il lavoro del revisore (altrimenti rischi di non avere revisioni o di aspettare parecchio) dall'altra la modifiche devono avere tutte la loro giustificazione, perchè infuturo, quando sarà necessario andare a vedere perchè è stata una fatta una certa modifica, sarà molto più difficile capirlo

Reply:
quindi ricapitolo, se si è partiti da un codice formattato diversamente non è possibile modificarne la formattazione anche se in senso migliorativo. Corretto?

Reply:
Certo che è possibile, fai una PR apposta o almeno un commit dedicato

Reply:
Nah, ci rinuncio ..non ne esco più .."lego l'asino dove vuole il padrone" cit.

Reply:
> quindi ricapitolo, se si è partiti da un codice formattato diversamente non è possibile modificarne la formattazione anche se in senso migliorativo. Corretto?

Direi che è aspetto relativo. In questo caso specifico per te potrebbe essere migliorativa la nuova formattazione, per un altro no. IMHO, ad esempio, era più leggibile l'originale.

Ci sono comunque buona notizie. Per avere uniformità nella formattazione, dalla 13.0 OCA ha introdotto questo:
https://odoo-community.org/blog/the-oca-blog-1/post/black-isort-pre-commit-97

Reply:
> Ci sono comunque buona notizie. Per avere uniformità nella formattazione, dalla 13.0 OCA ha introdotto questo:
> https://odoo-community.org/blog/the-oca-blog-1/post/black-isort-pre-commit-97

lo uso già anche per la 12.0